### PR TITLE
進捗報告

### PR DESCRIPTION
**Pull request by mre1ght**
https://github.com/tigerscave/eight-js-easy-apps/issues/21
自分ではどうしようもなかったので、ラファさんに、ペダル設定のボタンの多重選択NGを実装してもらいました。
レバー設定については、明日、自分でやってみます。

### CODE REVIEW

**Comment by mre1ght** [comment]
#21
I made lever-settings page and link.
What i want to ask you is disable edit-button when I select all "未選択” button.
After selecting all 未選択-button, I could put disable.
But, under the first situation, I can push the edit-button I don't want to push.

File: cranetmock/index.html [hunk 1]
```diff
@@ -127,6 +127,7 @@
   </div>
 
   <script type="text/javascript" src="main.js"></script>
+  <script type="text/javascript" src="sub.js"></script>
```
Comment (rhmtrz) [inline cranetmock/index.html]: what is sub.js? what do you mean by name `sub`

Comment (rhmtrz) [reply cranetmock/index.html]: I think it is better to use a more specific name, 
The name should describe the file content. But `sub` describes the relationship between two items.

File: cranetmocklever/main.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  }
+})
+
+leverList3.addEventListener('change', event => {
+  if(event.target.value === leverList2.value || event.target.value === leverList4.value || event.target.value === leverList5.value){
+    editButton.setAttribute('disabled', null)
+  } else {
+    editButton.removeAttribute('disabled')
+  }
+  localStorage.setItem(event.target.id, event.target.value);
```
Comment (rhmtrz) [inline cranetmocklever/main.js]: Do you need to save the data to local storage in any way?  

I think when there is two same selection,  it no need to save to local storage

Comment (rhmtrz) [reply cranetmocklever/main.js]: example: 
```
if (optionA == optionB) {
 // disable button 
} else {
  // active button 
  // save to local storage 
}

```

File: cranetmock/index.html (hunk 1 above)
Comment (mre1ght) [reply cranetmock/index.html]: Thank you for teaching how to name  sub.js.
Understood.

File: cranetmocklever/main.js (hunk 2 above)
Comment (mre1ght) [reply cranetmocklever/main.js]: Getting clear.
I fixed.
I'll push it again.

**Review by rhmtrz — APPROVED** [review]

**Comment by mre1ght** [comment]
const disableSaveButton = () => {
  if (editButton.textContent === 'カスタム設定を保存' && value === "未選択") {
    editButton.setAttribute('disabled', '');
  }
};
I tried but,  I couldn't implement disable the "カスタム設定を保存" button when any "未選択" option is selected.
未選択が一つでも選択してある場合、カスタム設定を保存ボタンを無効化したかったのですが、できませんでした。
また明日質問させてください。

**Comment by mre1ght** [comment]
```
const disableSaveButton = () => {
if (editButton.textContent === 'カスタム設定を保存' && value === "未選択") {
editButton.setAttribute('disabled', '');
}
};
```
```
editButton.addEventListener = ('change', () => {
  if (editButton.textContent === 'カスタム設定を保存' && leverList2Value === '未選択' || leverList3Value === '未選択' || leverList4Value === '未選択' || leverList5Value === '未選択') {
    editButton.setAttribute('disabled', '');
  }
});
```

I want to implement a disable function when I select a 未選択-button.But I can't. 
How should I write here? Or tell me what the problem is.

**Comment by mre1ght** [comment]
#25 
とりあえず、developブランチにあげました。

**Pull request merged**